doc/man3/EVP_KEYEXCH_free.pod

```diff
@@ -58,6 +61,11 @@ all activated providers in the library context I<libctx>, and for each
 of the implementations, calls I<fn> with the implementation method and
 I<data> as arguments.
 
+EVP_KEYEXCH_gettable_ctx_params() and EVP_KEYEXCH_settable_ctx_params() get
+a constant B<OSSL_PARAM> array that describes the names and types of key
```

Comment:
"FOO_gettable_params() ... get a constant..." no they don't!  They *return* something.

Sadly, I've now seen that this description is prevalent throughout our doc/man3 documentation.

I suggest replacing "get" with "returns" or "is used to get" (the latter is used in doc/man3/OSSL_PROVIDER.pod)

Reply:
Used return or returns..